tools/test/audio/process_test.m

```diff
@@ -139,18 +139,15 @@
 	fprintf('Number of skipped tests = %d\n', r.n_skipped);
 	fprintf('============================================================\n');
 
-	n_fail = r.n_fail;
```

Comment:
Please prefix all your commit titles with Tools: Test: So that reviewers and and other git log viewers know what these are about. These do not contain changes for audio components.